### ISSUE TITLE
IsAsymmetricDigraph causes seg fault

### ISSUE THREAD

**Issue by james-d-mitchell** [gap-packages/Digraphs#52]
The following example:

```
gap> gr := Digraph([[1], [2], [1 .. 3]]);;
gap> IsAntisymmetricDigraph(gr);
```

causes a segmentation fault in version 0.7.0 (and all previous versions where this function exists).

I am about to commit a pull request which fixes this.

**Comment by james-d-mitchell** [comment]
This is the issue causing the tests to fail in PR #51.

**Comment by wilfwilson** [comment]
Resolved by PR #53.
